devtools/highlight.py

```diff
@@ -0,0 +1,77 @@
+import sys
+from typing import Optional
+
+from .ansi import isatty
+from .prettier import env_bool
+
+if sys.platform == 'win32':  # pragma: no cover (windows)
```

Comment:
```suggestion
if sys.platform == 'win32':
```

Reply:
can you wrap this method in a function rather than in this sys test.

e.g. something like:

```py
def use_ansi_colours(file_):
    if sys.platform == 'win32':
        return use_ansi_colours_windows(file_)
    else:
        return use_ansi_colours_unix(file_)

def use_ansi_colours_windows(file_):
    ... this code

def use_ansi_colours_unix(file_):
    ... is isatty code
```

Reply:
Sure i refactored the sys test. 

The suggested removal of the `pragma`:
I changed the `pragma` and introduced `covdefaults`. This package lets you use platform specific `pragma`.